MyWeight/Extensions/UITableViewCell+ReuseIdentifier.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UITableViewCell+ReuseIdentifier.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import UIKit
+
+extension UITableViewCell {
+    /**
+     - returns: a UINib with the default Nib name (matches the class name).
+        If the nib file is not found, this will cause a exception crash.
+     */
+    static func nib() -> UINib {
+        return UINib(nibName: defaultNibName, bundle: nil)
+    }
+    
+    /**
+     Default nib name for this class (matches the class name).
+     */
+    static var defaultNibName: String {
+        return String(self)
+    }
+    
+    /**
+     Default reuse identifier for this class (matches the class name).
+     */
+    static var defaultReuseIdentifier: String {
+        return String(self)
+    }
+}
+
+extension UITableView {
+    
+    /**
+     Register a cell into the tableView.
+     Uses the defaultReuseIdentifier if the reuseIdentifier is not provided.
+     
+     - parameter cellClass: the class to register.
+     */
+    func registerCellNib(cellClass: UITableViewCell.Type, forCellReuseIdentifier reuseIdentifier: String? = nil) {
```

Comment:
This method (and the family) has a very similar signature of the main one, two parameters.
I would suggest to make more explicit in the signature what happens when `reuseIdentifier` is `nil`. We use generics here:
```
func registerCellNib<CellType where CellType: UITableViewCell>(cellClass: CellType.Type, forCellReuseIdentifier reuseIdentifier: String = CellType.defaultReuseIdentifier) {
        self.registerNib(cellClass.nib(), forCellReuseIdentifier: reuseIdentifier)
}
```

What do you think?